src/api/admin/rbac.py

```
@@ -0,0 +1,568 @@
+    row = await UserRepo(session).get_by_id(user_id)
+    if row is None:
+        raise HTTPException(404, {"error": "user_not_found"})
+    return await _user_to_out(session, row)
+
+
+@router.patch("/users/{user_id}", response_model=UserOut)
+async def update_user(
+    user_id: str,
+    body: UserPatch,
+    request: Request,
+    actor: User = Depends(require_permission("users:invite")),
```

Ruff (0.15.12)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)